pkg/build/pipelines/python/pipinstall.yaml

```diff
@@ -0,0 +1,15 @@
+name: Pip Install a Python package
+
+needs:
+  packages:
+    - busybox
```

Comment:
Should this also require `pip` as a dependency? Like here:
https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/build.yaml#L5

Now, having said that, I wonder how this should be handled with the python versioning. I know @smoser has been working on that, so tagging him here.

Reply:
I thought about this, but there are 3 different pip packages, perhaps we add that as an input, defaulting to the most recent one?

Reply:
I'm guessing that @dakaneye found https://github.com/wolfi-dev/os/pull/20318/files after doing this.  (thanks for being thorough).

I think right now I'd not add this pipeline.  We may or may not end up adding one in the future, but our use of pip and python package installation is going through some changes.